lib/shelley/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -1259,7 +1229,7 @@ _assignScriptRedeemers (toAlonzoPParams -> pparams) ti resolveInput redeemers tx
     -- | Finally, calculate and add the script integrity hash with the new
     -- final redeemers, if any.
     addScriptIntegrityHash
-        :: forall era. (era ~ Cardano.ShelleyLedgerEra Cardano.AlonzoEra)
+        :: forall e. (e ~ Cardano.ShelleyLedgerEra Cardano.AlonzoEra)
```

Comment:
I'm curious as to why you changed this. Does it clash with another `era` definition?

Reply:
Yes